### ISSUE TITLE
Be able to revert a wrongly appointed goal

### ISSUE THREAD

**Issue by elgervb** [tuvokki/pleague#34]

**Comment by elgervb** [comment]
I'm working on it ;-)

**Comment by tuvokki** [comment]
While you're at it @elgervb can you fix this also?
![image](https://cloud.githubusercontent.com/assets/181719/15609909/e3f30612-2422-11e6-9f56-a6a68b3c6490.png)

**Comment by mabertcloud** [comment]
I think the easiest fix is to delete the player "AnnieAnnieAnnieAnnieAnnie"

**Comment by tuvokki** [comment]
Max player name is already limited to 26 chars due to my poor css skills 😜

Maybe we should add an ELO-penalty per letter ....

**Comment by tuvokki** [comment]
This functionality does not work as advertised, it seems that the only player you can revert a goal from is the top-right most player (in the above screenshot 'Annie...')

Also, when I revert a goal from Annie when she has no goals left it will be extracted from the blue team.

@elgervb can you take care of this and update the pull-request?
Also please remove the (rather profane) todo's from your commit, since they have no bearing on this case, you can create a new issue to refactor the logic.

**Comment by elgervb** [comment]
Correct. I have a fix ready. Shall check iT in somewhere thuis weekend

**Comment by tuvokki** [comment]
That's good hoor :bowtie:

**Comment by tuvokki** [comment]
It does not seem to work on mobile. I tested it on an iPhone and the long-press did not change the buttons to wiggly-waggle and invert.

**Comment by tuvokki** [comment]
Ps. please refrain from todo's in parts of the code that do not involve this functionality

**Comment by tuvokki** [comment]
@elgervb it would be great if you can you take another look at this, there are two topics to look into:
~~1. when reverting a goal from the red team, the blue defender also gets one goal reverted~~
2. it does not work on mobile
1. has been fixed in 30397c67c10a8e981f3e592396c4e379f1245673
